packages/contracts/test/10_unit-testing/11_plugin.ts

```diff
@@ -2045,39 +2040,41 @@ describe('Multisig', function () {
 
         // Approve and try execution as Alice although the `minApprovals` threshold is not met yet.
         let tx = await plugin.connect(alice).approve(id, true);
-        await expect(
+        let rc = await tx.wait();
```

Comment:
is `rc` a name clear enough?

Reply:
Hm, I think `tx` and `rc` are quite common.